assets/js/persistence.js

```diff
@@ -171,6 +180,10 @@ function restoreChoices(mode) {
             $('#originalText').val(decodeURIComponent(getURLParam('q')));
         }
 
+        if(getURLParam('qP').length > 0) {
```

Comment:
Your example is fine IMO. Yeah, it's a big more repetitive but it prevents constants from being repeated multiple times which leads to later errors.